src/lib/types/experimental.ts

```diff
@@ -268,10 +267,6 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_REMOVE_UNSAFE_INLINE_STYLE_SRC,
         false,
     ),
-    addonUsageMetrics: parseEnvVarBoolean(
-        process.env.UNLEASH_EXPERIMENTAL_ADDON_USAGE_METRICS,
-        false,
-    ),
```

Comment:
Should we keep it for a while? Because we're changing the code, we might be introducing a bug, so perhaps wrapping this code https://github.com/Unleash/unleash/pull/8381/files#diff-0652df03aa5f9d5c19c8b62cdd0b378a44adae9a05cf20f638552cc71ceef680R97-R100 with the flag?

Reply:
Re-added it 👍